doc/reference/configuration/configuration_reference.rst

```diff
@@ -392,6 +392,293 @@ audit_log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_AUDIT_LOG_SYSLOG_SERVER
 
+..  _configuration_reference_compat:
+
+compat
+------
+
+The ``compat`` section defines values of the :ref:`compat <compat-module>` module options.
+
+..  NOTE::
+
+    ``compat`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`compat.binary_data_decoding <configuration_reference_compat_binary_decoding>`
+* :ref:`compat.box_cfg_replication_sync_timeout <configuration_reference_compat_replication_timeout>`
+* :ref:`compat.box_error_serialize_verbose <configuration_reference_compat_error_serialize>`
+* :ref:`compat.box_error_unpack_type_and_code <configuration_reference_compat_error_unpack>`
+* :ref:`compat.box_info_cluster_meaning <configuration_reference_compat_yaml_pretty>`
+* :ref:`compat.box_session_push_deprecation <configuration_reference_compat_session_push>`
+* :ref:`compat.box_space_execute_priv <configuration_reference_compat_space_execute>`
+* :ref:`compat.box_space_max <configuration_reference_compat_space_max>`
+* :ref:`compat.box_tuple_extension <configuration_reference_compat_tuple_extension>`
+* :ref:`compat.box_tuple_new_vararg <configuration_reference_compat_tuple_new>`
+* :ref:`compat.c_func_iproto_multireturn <configuration_reference_compat_iproto_multireturn>`
+* :ref:`compat.fiber_channel_close_mode <configuration_reference_compat_fiber_channel>`
+* :ref:`compat.fiber_slice_default <configuration_reference_compat_cluster_meaning>`
+* :ref:`compat.json_escape_forward_slash <configuration_reference_compat_json_escape>`
+* :ref:`compat.sql_priv <configuration_reference_compat_sql_priv>`
+* :ref:`compat.sql_seq_scan_default <configuration_reference_compat_sql_scan>`
+* :ref:`compat.yaml_pretty_multiline <configuration_reference_compat_yaml_pretty>`
+
+
+.. _configuration_reference_compat_binary_decoding:
+
+.. confval:: compat.binary_data_decoding
+
+    Defines how to store binary data fields in Lua after decoding:
```

Comment:
Let's use [the imperative mood](https://www.tarantool.io/en/doc/latest/contributing/docs/api/#language-of-the-general-description) as in other descriptions config/API descriptions:

```suggestion
    Define how to store binary data fields in Lua after decoding:
```